test/fixtures/v5/continue/left_destination.json

```diff
@@ -37,6 +37,7 @@
         "tr": "Destination 1 istikametinde sol yöne dön",
         "uk": "Поверніть ліворуч у напрямку Destination 1",
         "vi": "Quẹo trái về Destination 1",
+        "yo": "Tan-anapa osisi ọnaDestination 1",
```

Comment:
Should the destination name be tacked onto the end of “ọna” like this, or should there be a space in between?

Reply:
there is supposed to be a space between

Reply:
And this should be fixed right in source translation strings on Transifex :wink:

Reply:
I fixed the whitespace issues in Transifex and this PR.